src/toolbar/ToolbarClick.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { adjustColor } from './utility';
+
+const iconStyle = item => (
+  item.text ?
+    { color: adjustColor(item.color, item.enabled), marginRight: '5px' } :
+    { color: adjustColor(item.color, item.enabled) }
+);
+
+export const ToolbarClick = props => (
```

Comment:
You can destructure the props here

Reply:
This applies to more places in this PR

Reply:
> You can destructure the props here

Here it would complicate passing the props to `iconStyle`